fix(curriculum): delineate span elements in typography (#55348)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/14-responsive-web-design-22/learn-typography-by-building-a-nutrition-label/615f7de4487b64919bb4aa5e.md

---
id: 615f7de4487b64919bb4aa5e
title: Step 54
challengeType: 0
dashedName: step-54
---

# --description--

> After your last `.divider`, create a new `p` element with the text `Cholesterol 0mg 0%`. Wrap the text `Cholesterol` in a `span` element, and give that `span` element the `class` of `bold`. Wrap the text `0%` in another `span` element, with the `class` of `bold`.
> 
> Nest the `Cholesterol` `span` element and the text `0mg` in an additional `span` element for alignment.  This final part will nest a `span` element within a `span` element.

Reviewer: ```suggestion
After your last `.divider`, create a new `p` element with the text `Cholesterol 0mg 0%`. Then, wrap the text `Cholesterol` in a `span` element, `0%` in another, and give each of them a class of `bold`.

Finally, nest the `span` element containing the text `Cholesterol` along with the text `0mg`, in an additional `span` element for alignment.
```
